src/include/homestore/checkpoint/cp_mgr.hpp

```diff
@@ -154,6 +154,7 @@ class CPManager {
     superblk< cp_mgr_super_block > m_sb;
     std::vector< iomgr::io_fiber_t > m_cp_io_fibers;
     iomgr::timer_handle_t m_cp_timer_hdl;
+    std::atomic< bool > m_shutdown_initiated{false};
```

Comment:
Probably moving this to homestore instance could be more reasonable?
And set it from homestore::shtudown?

Reply:
Actually its for the cp_mgr destructor to notify the cp_flush to not initiate back to back CP.

Reply:
I understand that, but cp_flush can also get homestore inst to get shutdown initiated, right?

Reply:
As discussed renamed the cp_shutdown.